source/common/upstream/upstream_impl.h

```diff
@@ -507,12 +509,21 @@ class ClusterImplBase : public Cluster, protected Logger::Loggable<Logger::Id::u
   virtual void startPreInit() PURE;
 
   /**
-   * Called by every concrete cluster when pre-init is complete. At this point, shared init takes
-   * over and determines if there is an initial health check pass needed, etc.
+   * Called by every concrete cluster when pre-init is complete. At this point,
+   * shared init starts init_manager_ initialization and determines if there
+   * is an initial health check pass needed, etc.
    */
   void onPreInitComplete();
 
+  /**
+   * Called by every concrete cluster after all Sds api targets registered at SDS init manager are
```

Comment:
Remove references to SDS or make it clear that SDS is one of the things that init manager is used for. We are likely to add other things in the future.

Reply:
Removed references to SDS from comment. Thanks.